src/pykiso/lib/auxiliaries/udsaux/uds_server_auxiliary.py

```diff
@@ -294,6 +312,82 @@ def _dispatch_callback(self, received_uds_data: List[int]) -> None:
         callback_to_execute(received_uds_data, self)
         return
 
+    def _create_uds_data(self, coded_values: List[int]) -> List[int]:
```

Comment:
You call your method _create_uds_data btu when I read the docstring it mentionned : "Format a list of coded"
Maybe call it "format_uds_data" it will be more accurate based on the docstring

Reply:
changed to suggestion